src/ProducerEventProcessor/ProducerEventProcessor.cpp

```diff
@@ -270,6 +322,11 @@ void ProducerEventProcessorImpl::ProcessCallstackSampleAndTransferOwnership(
 
 void ProducerEventProcessorImpl::ProcessCaptureFinishedAndTransferOwnership(
     CaptureFinished* capture_finished) {
+  if (!thread_state_slice_tid_and_begin_timestamp_to_callstack_.empty()) {
+    ORBIT_ERROR(
+        "There are cached callstacks for thread state slices left in the cache after the capture "
```

Comment:
I don't think we should call this a "cache".

Reply:
Rephrased it.